### DISCUSSION TITLE
Consider renaming the binary-ninja cask to binaryninja-free

### DISCUSSION THREAD

**Discussion by xusheng6** [Homebrew/discussions#5771] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.4.8
ORIGIN: https://github.com/Homebrew/brew
HEAD: e78a0adb4f260913fffdc956cc8c20fb54109feb
Last commit: 2 days ago
Branch: stable
Core tap JSON: 26 Nov 05:37 UTC
Core cask tap JSON: 26 Nov 05:37 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 12
Homebrew Ruby: 3.3.6 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.6/bin/ruby
CPU: dodeca-core 64-bit arm_lobos
Clang: 16.0.0 build 1600
Git: 2.39.5 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 15.1.1-arm64
CLT: 16.1.0.0.1.1729049160
Xcode: 16.1
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  z3
  zstd

Warning: /usr/bin occurs before /opt/homebrew/bin in your PATH.
This means that system-provided programs will be used instead of those
provided by Homebrew. Consider setting your PATH so that
/opt/homebrew/bin occurs before /usr/bin. Here is a one-liner:
  echo 'export PATH="/opt/homebrew/bin:$PATH"' >> ~/.zshrc

The following tools exist at both paths:
  crc32
  openssl
  pip3
  python3
  rev
  xattr

Warning: Homebrew's "sbin" was not found in your PATH but you have installed
formulae that put executables in /opt/homebrew/sbin.
Consider setting your PATH for example like so:
  echo 'export PATH="/opt/homebrew/sbin:$PATH"' >> ~/.zshrc
```

### Description of issue

To start with, thx for making the binary-ninja cask (https://github.com/Homebrew/homebrew-cask/blob/master/Casks/b/binary-ninja.rb) which makes it easier for people to install our product.

I would like to discuss whether it is possible to rename it to binary-ninja-free, or even better, binaryninja-free, to avoid potential confusion? I do not know what is the implications of renaming a cask, e.g., will people be able to auto-update to the new one or do they have to manually reinstall it? 

If this idea is good for you, I can create a PR for it as well

### Relevant casks

binary-ninja

**Comment by p-linnane** [comment]
There are a few things to consider here to determine if we can/should proceed. Is the existing binary able to be upgraded to the full version if a license is purchased? Or would it require removing the existing binary and running a different installer?

**Reply by psifertex** [reply]
It would necessitate removing the previous one or installing a separate one side-by-side. It cannot be upgraded in-place. This is why we'd rather the cask be renamed.

**Reply by xusheng6** [reply]
Yup, the binaries for the free version and the paid version are different. It is not unlocked by a license file.

**Reply by p-linnane** [reply]
Thanks for the clarification. Yes, we can rename it then. https://github.com/Homebrew/homebrew-cask/pull/193586

**Reply by psifertex** [reply]
Thanks for the quick response!
